eng/pipelines/common/global-build-job.yml

```diff
@@ -160,6 +173,31 @@ jobs:
         displayName: Disk Usage after Build
         condition: always()
 
+    # Build coreclr runtime
+    - ${{ if eq(parameters.compilerName, 'gcc') }}:
+      - script: $(Build.SourcesDirectory)/src/coreclr/build-runtime$(scriptExt) ${{ parameters.buildConfig }} ${{ parameters.archType }} -ci $(compilerArg) $(_officialBuildParameter)
+        displayName: Build CoreCLR Runtime
+
+    # Build mono runtime
+    - ${{ if eq(parameters.compilerName, 'gcc') }}:
+      - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) mono $(compilerArg) -arch ${{ parameters.archType }} -c ${{ parameters.buildConfig }} $(_officialBuildParameter) -ci
```

Comment:
Would be better to specify all subsets to build via a single build script invocation, like `mono+libs.native+host.native`?